### PR TITLE
CORTX-33607 Support for CORTX K8s version 0.9.0

### PR DESCRIPTION
**Pull request by shailesh-vaidya**
# Problem Statement
- Add support for v0.9.0 of https://github.com/Seagate/cortx-k8s/ 

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [x] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [x] Jenkins pipelines used for testing - https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/sv_space/job/setup-cortx-cluster/198/ 

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: solutions/kubernetes/cortx-deploy-functions.sh [hunk 1]
```diff
@@ -225,16 +224,19 @@ function add_image_info() {
...
+    add_secondary_separator "Updating node info in solution.yaml"    
 
     pushd $SCRIPT_LOCATION/k8_cortx_cloud
-        count=1
+        if ! grep -q nodes solution.yaml; then
+        count=0
         for node in $(kubectl get nodes -o jsonpath="{range .items[*]}{.metadata.name} {.spec.taints[?(@.effect=='NoSchedule')].effect}{\"\n\"}{end}" | grep -v NoSchedule)
             do
-            i=$node yq e -i '.solution.nodes['$count'].node'$count'.name = env(i)' solution.yaml
+            i=$node yq e -i '.solution.storage_sets[0].nodes['$count'] = env(i) ' solution.yaml
```
Comment (gauravchaudhari02) [inline solutions/kubernetes/cortx-deploy-functions.sh:234]: Please remove space after `env(i)`

File: solutions/kubernetes/cortx-deploy-functions.sh [hunk 2]
```diff
@@ -225,16 +224,19 @@ function add_image_info() {
...
             count=$((count+1))
-        done
-        sed -i -e 's/- //g' -e '/null/d' solution.yaml
+            done
+        else
+        add_secondary_separator "Nodes entries already present in solution.yaml. Skipping update"
```
Comment (gauravchaudhari02) [inline solutions/kubernetes/cortx-deploy-functions.sh:238]: please indent line 237-239 correctly

File: solutions/kubernetes/cortx-deploy-functions.sh (hunk 1 above)
Comment (shailesh-vaidya) [reply solutions/kubernetes/cortx-deploy-functions.sh]: Fixed and tested - https://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/RGW/job/setup-cortx-rgw-cluster/9172/console

File: solutions/kubernetes/cortx-deploy-functions.sh (hunk 2 above)
Comment (shailesh-vaidya) [reply solutions/kubernetes/cortx-deploy-functions.sh]: Fixed and tested - https://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/RGW/job/setup-cortx-rgw-cluster/9172/console

**Comment by shailesh-vaidya** [comment]
@gauravchaudhari02 - Please review this PR

**Review by gauravchaudhari02 — APPROVED** [review]

**Pull request merged**